markets/perps-market/contracts/utils/OffchainUtil.sol

```diff
@@ -44,11 +44,11 @@ library OffchainUtil {
         updateData[0] = result;
 
         IPythVerifier verifier = IPythVerifier(settlementStrategy.priceVerificationContract);
-        uint256 msgValue = verifier.getUpdateFee(1);
+        uint256 msgValue = verifier.getUpdateFee(updateData);
```

Comment:
@leomassazza  or @sunnyvempati  I'd expect the keeper that settles the transaction to pass along the right fee on `msg.value`. Why are we looking it up?